common/hashing.h

```diff
@@ -172,33 +204,46 @@ inline auto HashValue(const T& value) -> HashCode;
 // sizes are compared as part of equality comparison, then that dynamic size
 // should typically be included in the hash.
 //
-// It is essential that values that compare equal have equal hashes, and
-// desirable that values that compare unequal have a high probability of
-// different hashes.
+// This type's API also reflects the reality that high-performance hash tables
+// rely on keys that are generally small and cheap to hash. The result is
```

Comment:
```suggestion
// are used with keys that are generally small and cheap to hash. The result is
```
Remaining sentences in this paragraph are adding very little and could probably just be dropped.

Reply:
Sure.